website/pages/docs/agent/telemetry.mdx

```diff
@@ -137,44 +137,46 @@ when retrieving metrics from the built-in store using the above described signal
+| `consul.client.api.catalog_node_services.`            | This increments whenever a Consul agent receives a request to list services registered in a node.                                                                                                                                                                                                                                                                                                                   | requests             | counter |
+| `consul.client.api.success.catalog_node_services.`    | This increments whenever a Consul agent successfully responds to a request to list services in a node.                                                                                                                                                                                                                                                                                                           | requests             | counter |
+| `consul.client.rpc.error.catalog_node_services.`      | This increments whenever a Consul agent receives an RPC error for a request to list services in a node.                                                                                                                                                                                                                                                                                                          | errors               | counter |
+| `consul.client.api.success.catalog_gateway_services.` | This increments whenever a Consul agent successfully responds to a request to list services associated with a gateway.                                                                                                                                                                                                                                                                                                           | requests             | counter |
+| `consul.client.rpc.error.catalog_gateway_services.`   | This increments whenever a Consul agent receives an RPC error for a request to list services associated with a gateway.                                                                                                                                                                                                                                                                                                          | errors               | counter |
```

Comment:
We have mismatched docs and implementations here, I think we probably want 3 metrics here, like `catalog_node_services`:
1. `consul.client.api.catalog_gateway_services.`
2. `consul.client.api.error.catalog_gateway_services.`
3. `consul.client.api.sucess.catalog_gateway_services.`

The docs have `success` and `error`, while the implementation has `consul.client.api.catalog_gateway_services.` and `error`.